README.md

```diff
@@ -9,7 +9,8 @@ It supports all features of TypeScript including type-checking. [Read more about
 ---
 
 | We are not doing semantic versioning and `23.10` is a re-write, run `npm i -D ts-jest@"<23.10.0"` to go back to the previous version |
-|---|
+| ------------------------------------------------------------------------------------------------------------------------------------ |
```

Comment:
why do we have this change ?

Reply:
I think is auto-formatted by prettier. I will edit this back to this `|---|`.